medicines/api/src/product.rs

```diff
@@ -25,6 +26,28 @@ impl Product {
     }
 }
 
+#[juniper::graphql_object]
+#[graphql(description = "A medical product containing active ingredients")]
+impl Product {
+    fn name(&self) -> &str {
+        &self.name
+    }
+    fn documents(&self, first: Option<i32>, skip: Option<i32>) -> document::Documents {
+        let docs = self.documents.clone().into_iter();
+        let docs = match first {
+            Some(t) => docs.take(t as usize).collect(),
+            None => docs.collect(),
+        };
+
+        let offset = match skip {
+            Some(a) => a,
+            None => 0,
+        };
```

Comment:
It's perhaps trying to be a bit too brief but you could do `skip.unwrap_or_default()` or `skip.unwrap_or(0)` instead?

Reply:
yeah, either is fine really, I've just tried to get away from `unwrap_or` because it's enforcing a bad habit of writing `unwrap` in production code